src/app/shared/components/common/info-box/info-box.component.html

```diff
@@ -3,7 +3,8 @@
     *ngIf="editRouterLink"
     class="float-right btn-tool"
     [routerLink]="editRouterLink"
-    [title]="'checkout.address.update.label' | translate"
+    [title]="('checkout.address.update.label' | translate) + ' ' + (heading | translate)"
+    [attr.aria-label]="('checkout.address.update.label' | translate) + ' ' + (heading | translate)"
```

Comment:
- Only works in English, not for e.g German 
- should be discussed to only use the aria-label because screen readers aren't consistent when it comes to reading titles, but when they do it'd read both title and aria-label

Reply:
has been fixed to work with every language

Reply:
decided to remove the aria-label and just use the title so it is consistent to other icon-elements